WORKSPACE

```diff
@@ -56,9 +56,9 @@ load("//bazel_tools:build_environment.bzl", "build_environment")
 
 build_environment(name = "build_environment")
 
-load("//bazel_tools:scala_version.bzl", "scala_version")
+load("//bazel_tools:scala_version.bzl", "scala_version_rule")
```

Comment:
Happy to come up with a new name, but I'm not sure what. The previous name was a problem because it collided with another `scala_version` in the same file, and Skylark shadowing rules confuse the hell out of me.

Reply:
hm naming is hard, @aherrmann-da any ideas?

Reply:
`scala_version_configure` perhaps. The `_configure` suffix is commonly used in repository rules that set up toolchains. Not exactly the same thing, but close.